default.nix

```diff
@@ -1,5 +1,7 @@
 let
   localLib = import ./lib.nix;
+  walletConfigFile = ./custom-wallet-config.nix;
```

Comment:
(nitpick) I suggest we move this down, inside to `connect` function with a new `let .. in`